index.js

```diff
@@ -0,0 +1,14 @@
+// implement a fizz buzz function.
+
+function fizzBuzz(n) {
+  var result = []
```

Comment:
this should be a const
```suggestion
  const result = []
```

Reply:
Here are some code suggestions:
```suggestion
  helloWorld();
```